docs/user_guides/fs/feature_group/data_validation.md

```diff
@@ -6,25 +6,29 @@ Clean, high quality feature data is of paramount importance to being able to tra
 
 ## UI
 
-### Step 1: Create or Edit a Feature Group
+### Create a Feature Group (Pre-requisite)
 
-You can attach at most one expectation suite to a Feature Group. It can be done on creation or at a later point in time. Data validation is an optional step and is not required to write to a Feature Group. You can find out more information about creating a Feature Group [here](create.md).
+In the UI, one must create a Feature Group first before attaching an Expectation Suite. You can find out more information about creating a Feature Group [here](create.md). You can attach at most one expectation suite to a Feature Group. Data validation is an optional step and is not required to write to a Feature Group.
 
-Click on the Feature Group section in the navigation menu. Click on `New Feature Group` if you want to create a brand new Feature Group. If you already created your Feature Group you can use the search bar to find and open it. Select `edit` at the top or scroll to Expectations section and click on `Edit Expectations`.
+### Step 1: Find and Edit Feature Group
+
+Click on the Feature Group section in the navigation menu. Find your Feature Group in the list and click on its name to access the Feature Group page. Select `edit` in the top right corner or scroll to the Expectations section and click on `Edit Expectation Suite`.
 
 ### Step 2: Edit General Expectation Suite Settings
 
 Scroll to the Expectation Suite section. You can pick a name for your suite as well as two general options.
 
-- enabled checkbox controls whether validation will be run automatically before writing a DataFrame to a Feature Group. Note that validation is executed by the client.
+- Checkbox enabled. It controls whether the Expectation Suite will be used to validate a Dataframe automatically upon insertion into a Feature Group. Note that validation is executed by the client. Disabling validation allows to skip the validation step without deleting the Expectation Suite.
```

Comment:
It controls whether -> This controls whether

Classic french speaker mistake - "this allows to ..."
Disabling validation allows you to skip

(active voice)